### DISCUSSION TITLE
Comparison with Scrapy

### DISCUSSION THREAD

**Discussion by AhmedThahir** [rushter/selectolax#75] [Q&A] [answered]
How does selectolax compare with Scrapy with regards to performance of HTML parsing?

**Comment by rushter** [comment, marked as answer]
Internally, Scrapy uses lxml.

**Comment by AhmedThahir** [comment]
Since selectolax is faster than lxml, is there a way to use selectolax with scrapy?
Can you share some examples? (I couldn't find any)

**Reply by rushter** [reply]
No, currently, there is no way to use selectolax with scrapy. 

https://github.com/scrapy/parsel/issues/239
